solr/core/src/java/org/apache/solr/servlet/PrometheusMetricsServlet.java

```diff
@@ -83,6 +88,29 @@ static void writeStats(PrintWriter writer) {
     UnixOperatingSystemMXBean osBean = (UnixOperatingSystemMXBean) ManagementFactory.getOperatingSystemMXBean();
     writeProm(writer, "open_file_descriptors", PromType.gauge, "the number of open file descriptors on the filesystem", osBean.getOpenFileDescriptorCount());
 
+    writeCacheMetrics(writer);
     writer.flush();
   }
+
+  private static void writeCacheMetrics(PrintWriter writer) {
+    Supplier<Map> supplier = (Supplier<Map>) SolrDispatchFilter.instance.cores.getZkController().getSolrCloudManager().getObjectCache().get("fs-shared-caches");
```

Comment:
Where does this `fs-shared-caches` constant come from? I don't see other refs to this name in git.

Reply:
`fs-shared-caches` is the name by which the cache stats are stored in a global registry

Reply:
@dragonsinth these cache stats are for the new caffeine cache (back ported from Solr 8 to Solr 7) that we will be using for some new internal work until we make the switch to 8. I sent a note in slack with links to the relevant PRs.

Reply:
Agreed though we probably want to make this string a public static constant. And reference it at the publish site (provided the class is importable).